manifest.xml

```diff
@@ -9,5 +9,4 @@
   <logo>http://</logo>
   <depend package="base/cmake" />
   <depend package="base/types" />
-  <depend package="drivers/hidapi" />
```

Comment:
(Which by the way could have been called drivers/hidapi as well ... but I digress)

Then you should add the osdep dependency as well, right now you're only removing it.